File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.py
class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        max_freq = max(freq for n, freq in Counter(nums).items())

        res = 0
        for n, freq in Counter(nums).items():
            if freq == max_freq:
                res += freq
        return res

File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.cpp
class Solution {
public:
    int maxFrequencyElements(vector<int>& nums) {
        unordered_map<int, int> mp;
        int max_freq = 0, res = 0;
        for (int n : nums) {
            mp[n]++;
            max_freq = max(max_freq, mp[n]);
        }

        for (const auto& [k, v] : mp) {
            if (v == max_freq) {
                res += v;
            }
        }

        return res;
    }
};